contrib/largeNbDicts/largeNbDicts.c

```diff
@@ -697,7 +697,7 @@ static int benchMem(slice_collection_t dstBlocks, slice_collection_t srcBlocks,
             BMK_createTimedFnState(total_time_ms, ms_per_round);
 
     decompressInstructions di = createDecompressInstructions(ddictionaries);
-    compressInstructions ci;
+    compressInstructions ci = {0};
```

Comment:
Thanks for the review. In [this commit](https://github.com/facebook/zstd/commit/d0c88afe6d5c0a29274df735a044154c6bf6deaa) I changed the program to call `createCompressInstructions` only when we're benching compression, because that function calls `ZSTD_CCtx_setParametersUsingCCtxParams` and `cctxParams` is NULL during decompression, resulting in a segfault. I think I can change this to always call `createCompressInstructions` and check if `cctxParams` is NULL in that function instead.

Reply:
Sounds good to me.